src/main.rs

```diff
@@ -104,21 +102,6 @@ fn setup_logging(verbosity_level: log::Level) -> Result<()> {
     Ok(())
 }
 
-fn create_databroker_endpoint(host: String, port: u64) -> Result<Endpoint> {
-    let databroker_address = format!("{}:{}", host, port);
-
-    let endpoint = tonic::transport::Channel::from_shared(databroker_address.clone())
-        .with_context(|| "Failed to parse server url")?;
-    let endpoint = endpoint
-        .initial_stream_window_size(1000 * 3 * 128 * 1024) // 20 MB stream window size
```

Comment:
Wouldn't it be useful to have these endpoint settings configurable by the user with additional command line options?

We already know that you might want to fine-tune these settings depending on the available memory on the device and the expected bus load.

Could be separate feature/story, just a hint for the future.